src/redux/actions/authActions.js

```diff
@@ -70,3 +71,21 @@ export const registerAccount = (userData, history) => async (dispatch) => {
     return dispatch(authFailed(message));
   }
 };
+
+export const logout = () => ({
+  type: LOGOUT,
+});
+
+export const logoutAccount = history => async (dispatch) => {
+  try {
+    await API_SERVICE.post('/auth/logout');
+    alert.success('You have successfully signed out!');
```

Comment:
I think this alert should come after the token has been removed from local storage

Reply:
Okay I'll do that

Reply:
done